pkg/plugin/client/client.go

```diff
@@ -85,4 +98,6 @@ type Provider interface {
 type PluginInstance struct {
 	plugin.Plugin
 	Provider
+	CustomCommands        map[string]*plugin.Command
```

Comment:
@f0rmiga this is the kind of info I thought would end up in here. WDYT?

Reply:
Let's get this in, then we can think again about how to split this struct.